include/deal.II/grid/grid_generator.h

```diff
@@ -1188,14 +1188,22 @@ namespace GridGenerator
    * @param n_cells_toroidal Optional argument to set the number of cell
    * layers in toroidal direction. The default is 6 cell layers.
    *
+   * @param phi Optional argument to generate an open torus with angle
+   * $0 < \varphi <= 2 \pi$. The default value is $2 \pi$,
+   * in which case a closed torus is generated. If the torus is open,
+   * the torus is cut at two planes perpendicular to the torus centerline.
+   * The center of these two planes are located at $(x_1, y_1, z_1) = (R, 0, 0)$
+   * and $(x_2, y_2, z_2) = (R \cos(\varphi), 0, R \sin(\varphi))$.
```

Comment:
We should document the boundary ids assigned to the three different boundary parts.

Reply:
The boundary IDs should all be 0 (which is the deal.II default value?), so I would not document

Reply:
You're right: I misread the code that checks face numbers as checking boundary ids, so I thought the new boundaries had ids 4 and 5.